ONSdigital/design-system
# Update colour palette to align with ONS brand guidelines

### What is the context of this PR?
New ONS brand guidelines have been published, with an updated and consolidated colour pattern, intended to work across both DST and DP: https://www.notion.so/Colours-9cd14589e0764e33a3b3c5a6762f37ae

This PR is to update the colour palette to align both colour values and names to match these new guidelines.

### How to review
- Check the newly assigned variables 
- Check colours style documentation
- Check contrast ratios
